crates/ethereum/evm/src/lib.rs

```diff
@@ -151,6 +140,7 @@ impl ConfigureEvmEnv for EthEvmConfig {
     type Transaction = TransactionSigned;
     type Error = Infallible;
     type TxEnv = TxEnv;
+    type Spec = revm_primitives::SpecId;
```

Comment:
this makes perfect sence, 
with new revm this will be network specific so exposing this is a good call